src/Step/Deploy/PreDeploy/CleanRedisCache.php

```diff
@@ -74,12 +74,10 @@ public function execute()
             }
             $redisConfig = $cacheConfig['backend_options'];
             $this->logger->info("Clearing redis cache: $cacheType");
-            $cmd = 'redis-cli';
-            $cmd .= isset($redisConfig['server']) ? ' -h ' . $redisConfig['server'] : '';
-            $cmd .= isset($redisConfig['port']) ? ' -p ' . $redisConfig['port'] : '';
-            $cmd .= isset($redisConfig['database']) ? ' -n ' . $redisConfig['database'] : '';
-            $cmd .= ' flushdb';
-            $this->shell->execute($cmd);
+            $redisClient = new \Redis();
```

Comment:
Maybe will be even better to use PHP implementation of Redis client like colinmollenhour/credis.

Reply:
If its ok to add this dependency, I would absolutely support it and change this PR accordingly.

Reply:
@kandy ^ please review. I assume it's okay given Magento core anyway uses it? Not sure if we even need to explicitly add the dependency here or just let core decide. I would assume we want some fallback in this case too?

Reply:
Magento already have dependencies on `colinmollenhour/credis` from 1.6 - 1.10
So, I think it's ok to add it to ece-tools as well as  `"colinmollenhour/credis": "1.*"`